playbooks/ansible-test-cloud-integration-base/run.yaml

```diff
@@ -14,6 +14,18 @@
         _test_options: "{{ _test_options }} --no-temp-workdir"
       when: zuul.projects['github.com/ansible/ansible'].checkout not in ["stable-2.6", "stable-2.7"]
 
+    - name: "wait for the vcenter service"
```

Comment:
this runs from zuul-executor, which cannot connect to the appliance directly.

Reply:
actually, this is something we should test. Maybe zuul-executor will have correct SSH info, after we create it.

Reply:
It's an HTTP call.